File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// Pokemon3d SDK exists test

require_once __DIR__ . '/../pokemon3d_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = Pokemon3dSDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}

File: rb/test/exists_test.rb
# Pokemon3d SDK exists test

require "minitest/autorun"
require_relative "../Pokemon3d_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = Pokemon3dSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
